### PR TITLE
Add contribution guide.

### PR DESCRIPTION
**Pull request by sudeeptarlekar**
Added contribution guide for new user.

### CODE REVIEW

**Comment by sudeeptarlekar** [comment]
@DmitryAstafyev can you please review.
cc: @marcmo 

Thanks in advance

File: contribution.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+We are using [yarn](https://yarnpkg.com/) as a package manager. Installing yarn is simple
+as you just have to run following command in the terminal.
+
+```
+npm install -g yarn
+```
+
+## Build
+As mentioned earlier backend is built in Rust which takes care of reading the log files
+and frontend app is built using Electron JS.
```
Comment (DmitryAstafyev) [inline contribution.md:77]: I would add here, something like
### Completed build / rebuild
Completed build/rebuild could be done with 
```
rake developing:clean_rebuild_all
```
We are recommend use this command for the first time of building and also in cases when instance of chipmunk was updated/rebased.

File: contribution.md [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+
+### Changes in both backend and frontend
+```
+cd application/holder
+rake developing:holder_platform_bindings
+yarn run electron
+```
+
+With this you can take a look at the open issues and open a PR to fix them.
+
```
Comment (DmitryAstafyev) [inline contribution.md:122]: Probably make sense to add here something like:
### Build/developing commands
Full list of commands you can see with 
```
rake -T
```
Also you can use interactive mode with starting `rake` without any arguments
Peek 2023-04-10 10-09

**Review by DmitryAstafyev — CHANGES_REQUESTED** [review]
In general make sense to add section about checking of code. Like "before creating of pull-request you should:":
- make sure, there are no issues related to `TypeScript`: `rake lint:all`. This command will check all `TypeScript` related parts of application
- make sure, there are no issues related to `Rust`: `rake clippy:all`. This command will check of `Rust` related parts of application
- make sure, basic tests are good with `rake test:all`

**Review by DmitryAstafyev — APPROVED** [review]

**Pull request merged**